Pipfile

```diff
@@ -4,37 +4,38 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-aiodns = "~=2.0.0"
-aiohttp-jinja2 = "~=1.2.0"
-aiohttp-session = {version = "~=2.9.0", extras = ["aioredis"]}
+aiohttp-jinja2 = "~=1.5"
+aiohttp-session = {version = "~=2.11.0", extras = ["aioredis"]}
 aiohttp-utils = "~=3.1.1"
-aiohttp = "~=3.7.4"
-aioredis = "~=1.3.1"
-babel = "==2.9.1"
-cchardet = "~=2.1.6"
+aiohttp = "~=3.8.1"
+aioredis = "~=2.0.1"
+babel = "==2.10.1"
+cchardet = "~=2.1.7"
```

Comment:
I think `cchardet` is unused, tested locally and unit/cucumber tests pass without it